yarn.lock

```diff
@@ -2761,16 +2761,32 @@ color-name@1.1.3:
   resolved "https://registry.yarnpkg.com/color-name/-/color-name-1.1.3.tgz#a7d0558bd89c42f795dd42328f740831ca53bc25"
```

Comment:
You cannot commit lock file without package json when a dependency is added (in this case `sharp` I guess)